include/keisan/matrix/matrix.hpp

```diff
@@ -78,6 +78,10 @@ class Matrix
   const double * operator[](size_t pos) const;
 
   bool inverse();
+  bool inverse2();
+
+  Matrix<N, M> transpose() const;
+  Matrix<M, N> round(const double & tolerance) const;
```

Comment:
Pass `tolerance` as `double` directly instead:
```suggestion
  Matrix<M, N> round(double tolerance) const;
```